packages/odyssey-react-mui/src/Radio.tsx

```diff
@@ -41,36 +47,66 @@ export type RadioProps = {
    * Callback fired when the state is changed. Provides event and checked value.
    */
   onChange?: MuiRadioProps["onChange"];
+  /**
+   * Callback fired when the blur event happens. Provides event value.
+   */
+  onBlur?: MuiFormControlLabelProps["onBlur"];
 } & Pick<FieldComponentProps, "isDisabled" | "name"> &
-  SeleniumProps;
+  AllowedProps;
 
 const Radio = ({
+  inputFocusRef,
```

Comment:
Is the plan that any bi-directional props (like `blur()` and `focus()` are gonna have separate ref props?

If not, we should instead make this more generic so consumers only have to worry about creating a single ref to pass.

I would rename this more generically such as `inputRef`.